learn-test.gemspec

```diff
@@ -26,9 +26,10 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "oj", "~> 2.9"
   spec.add_runtime_dependency "faraday", "~> 0.9"
   spec.add_runtime_dependency "crack", "~> 0.4.3"
-  spec.add_runtime_dependency "jasmine", "~> 2.6.0", '>= 2.6.0'
+  spec.add_runtime_dependency "jasmine", "~> 2.6.0", ">= 2.6.0"
```

Comment:
Aren't the `>=` version locks on this, webrick, and selenium superfluous? Won't the twiddle-waka take precedence b/c it's more restrictive?

Reply:
yeah probably, down to change this if that's the case. didn't need to change to get this fix out.